.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
self-serve portal also has:
```
"printWidth": 100,
"singleQuote": true,
```

Is it worth having these as well?

Reply:
I've seen we've got both single quote and double quote in this project, I think it makes sense to bring this in settings. Just updated now